tests/torch/test_extractor.py

```diff
@@ -125,9 +125,10 @@ def tes_extract_model_for_node_with_fq(model_cls, input_node_name, output_node_n
     with torch.no_grad():
         ret1 = q_model(example_input)
         ret2 = extracted_module(example_input)
-        assert torch.any(torch.isclose(ret1, ret2))
+        assert torch.all(torch.isclose(ret1, ret2))
+
+    extracted_fn = extracted_module
+    if isinstance(extracted_fn, nn.Sequential):
+        extracted_fn = extracted_module[0]
 
-    if isinstance(extracted_module, nn.Sequential):
-        assert extracted_module[0].w_fq is not None
-    else:
-        assert extracted_module.w_fq is not None
```

Comment:
@AlexanderDokuchaev, this test was disable due to typo in the func name, please confirm I fixed it correctly

Reply:
yes